docs/guide/multisite.md

```diff
@@ -2,109 +2,202 @@
+Add as many Location pathStrings as you like.
+In the example above, the Media and Images folders will be accessible using their own URI even though they are not below `content.tree_root.location_id`.
 
-## Usage
+!!! note
 
-### Multisite Design
+    Leading slashes (`/`) are automatically trimmed internally, so they can be ignored.
 
-eZ Platform does not apply a [Legacy template fallback](https://doc.ez.no/display/EZP/Legacy+template+fallback) like eZ Publish did. You can, however, have different designs in your multisite installation if you organize the view configuration with the use of siteaccesses.
+## Configuration example
 
-You can see how this is done on the example of a multisite that contains two blogs with different designs, but sharing some templates. Let's assume you have a siteaccess called `second_site`. You can organize your templates in the following folder structure:
+To see how multisite can be used, let's look at an example of two sites using the same eZ Platform instance: a general site and a children's site.
+Separate SiteAccess groups are set up for the two sites:
 
 ``` yaml
-views
- |- pagelayout.html.twig
-     |- full
-         |- blog.html.twig
-         |- blog_post.html.twig
-     |- second_site
-         |- pagelayout.html.twig
-         |- full
-             |- blog_post.html.twig
+ezpublish:
+    siteaccess:
+        list: [site, kids]
+        groups:
+            site_group: [site]
+            kids_group: [kids]
+        default_siteaccess: site
+        match:
+            URIElement: 1
 ```
 
-Now you can use this view configuration (stored e.g. in a `views.yml` file):
+This is your content structure:
+
+```
+├── Content
+│   ├── General
+│   │   └── Articles
+│   │       └── Mammals
+│   └── Children
+│       └── Articles
+│           └── Lions
+└── Media
+    └── Images
+        └── Animal-photos
+```
+
+You can now set the root level for `kids_group` to only access the "Children" Location and its sub-items:
 
 ``` yaml
 ezpublish:
     system:
-        default:
-            pagelayout: pagelayout.html.twig
-            content_view:
-                full:
-                    blog:
-                        template: "full/blog.html.twig"
-                        match:
-                            Identifier\ContentType: [blog]
-                    blog_post:
-                        template: "full/blog_post.html.twig"
-                        match:
-                            Identifier\ContentType: [blog_post]
-        second_site:
-            pagelayout: second_site/pagelayout.html.twig
-            content_view:
-                full:
-                    blog_post:
-                        template: "second_site/full/blog_post.html.twig"
-                        match:
-                            Identifier\ContentType: [blog_post]
+        kids_group:
+            content:
+                tree_root:
+                    # LocationId of "Children"
+                    location_id: 57
```

Comment:
Root + children could lead to confusion (see my previous comment about children's site)